#include <exception>

class OutOfRange : public std::exception {

};
